src/table.rs

```diff
@@ -469,3 +469,353 @@ impl Table {
         self.inner.max_version()
     }
 }
+
+#[cfg(test)]
```

Comment:
How about using separate file so that the current one won't get too long? You can do that by creating file src/table/tests.rs.

Reply:
Alright. But we may have to export some functions as `pub(crate)` in this case.

Reply:
I don't think so. Access level is based on model instead of files.

Reply:
I've moved that to a separate file.